lib/auth/windows/windows.go

```diff
@@ -64,6 +80,10 @@ func getCertRequest(req *GenerateCredentialsRequest) (*certRequest, error) {
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
+	createUser, err := createUsersExtension(req.CreateUser, req.Groups)
```

Comment:
Do you know if there is any problem providing empty values for `CreateUser` and `Groups`? I'm asking because this function is also used to generate [MSSQL certificate requests](https://github.com/gravitational/teleport/blob/master/lib/auth/windows/windows.go#L187), and I'm not sure if we want to include them, but in the case where nothing happens when they're empty, it's ok to have them. Otherwise, we would need to check if they're not empty before adding them to the CSR.

Reply:
I've modified the code to only include extension when when we do have `true` value for `CreateUser` (on auth package side no extension is the same as extension with value `false`). It will now be transparent to DB cert request.